plugins/action/zos_job_submit.py

```diff
@@ -24,6 +24,11 @@
 from ansible_collections.ibm.ibm_zos_core.plugins.module_utils import template
 
 
+display = Display()
+
+from ansible_collections.ibm.ibm_zos_core.plugins.module_utils import template
```

Comment:
This import and the line above it are duplicates squeezed between the original code

Reply:
done